webknossos/webknossos/dataset/layer.py

```diff
@@ -168,12 +168,12 @@ def _get_sharding_parameters(
 
 class Layer:
     """
-    A `Layer` consists of multiple `webknossos.dataset.mag_view.MagView`s, which store the same data in different magnifications.
+    A `Layer` consists of multiple `MagView`s, which store the same data in different magnifications.
     """
 
     def __init__(self, dataset: "Dataset", properties: LayerProperties) -> None:
         """
-        Do not use this constructor manually. Instead use `webknossos.dataset.layer.Dataset.add_layer` to create a `Layer`.
+        Do not use this constructor manually. Instead use `Dataset``.add_layer()` to create a `Layer`.
```

Comment:
```suggestion
        Do not use this constructor manually. Instead use `Dataset.add_layer()` to create a `Layer`.
```

Reply:
The first version has the benefit, that `Dataset` is linked, in the second nothing is linked. Not sure which to prefer.